### PR TITLE
Awaiting go live: DOCS-3371: Amazon Pay

### PR DESCRIPTION
**Pull request by Sarah-MultiSafepay**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

File: content/payment-methods/amazon-pay.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+<details id="how-to-activate-amazon-pay"> 
+<summary>How to activate Amazon Pay</summary>
+<br>
+
+1. Email a request to <risk@multisafepay.com> 
+2. We check your eligibility and if approved, activate the payment method for your account. 
+3. Once approved, sign in to your <a href="https://merchant.multisafepay.com" target="_blank">MultiSafepay dashboard</a> <i class="fa fa-external-link" style="font-size:12px;color:#8b929e"></i>.
+4. Go to **Settings** > **Amazon** > **Merchant registration**. You are redirected to Amazon Pay's <a href="https://pay.amazon.com/signup" target="_blank">business registration</a> <i class="fa fa-external-link" style="font-size:12px;color:#8b929e"></i> to create an Amazon Payments merchant account.
+5. Enter the required information to create an account. Amazon sends you a confirmation email.
```
Comment (FrancescaNCook) [inline content/payment-methods/amazon-pay.md:60]:
```suggestion
5. Enter the required information to create an account. 
    Amazon sends you a confirmation email.
```

File: content/payment-methods/amazon-pay.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+> **Support:** If the payment method isn't visible in your dashboard, email <integration@multisafepay.com>
```
Comment (FrancescaNCook) [inline content/payment-methods/amazon-pay.md:62]:
```suggestion
**Support:** If the payment method isn't visible in your dashboard, email <integration@multisafepay.com>
```

File: content/payment-methods/chargebacks.md [hunk 3]
```diff
@@ -33,6 +33,12 @@ Card schemes generally allow cardholders to request chargebacks for up to 180 da
 
 By offering credit card payment methods, you agree to the cardholder rights guaranteed by the card schemes.
 
+# Amazon Pay chargebacks
+
+If an Amazon customer requests a chargeback from their bank or credit card issuer for an Amazon Pay payment, Amazon Pay will notify you by email. If you do not respond within 11 calendar days, then Amazon Pay will automatically debit the chargeback amount from your Amazon Payments merchant account.
```
Comment (FrancescaNCook) [inline content/payment-methods/chargebacks.md:38]:
```suggestion
If an Amazon customer requests a chargeback from their bank or credit card issuer for an Amazon Pay payment, Amazon Pay notifies you by email. If you do not respond within 11 calendar days, then Amazon Pay automatically debits the chargeback amount from your Amazon Pay merchant account.
```

File: content/payment-methods/google-pay-integration.md [hunk 4]
```diff
@@ -45,7 +45,7 @@ Add the Google Pay JavaScript library to the bottom of the `<body>` of your chec
   onload="onGooglePayLoaded()">
 </script>
 ```
-**Note:** The `onload` event handler is used to display the **Google Pay** button. For more information, see [Display the Google Pay button](#display-the-google-pay-button).
+**Note:** The `onload` event handler is used to display the **Google Pay** button. For more information, see [Display the Google Pay button](#3-display-the-google-pay-button).
```
Comment (FrancescaNCook) [inline content/payment-methods/google-pay-integration.md:48]: Should this be camel case? `onLoad`?

File: content/payment-methods/testing.md [hunk 5]
```diff
@@ -541,6 +541,19 @@ Any other card number receives an "Invalid card number" error.
 
 </details>
 
+<details id="amazon-pay">
+<summary>How to test Amazon Pay</summary>
+<br>
+
+1. [Create an order](/reference/createorder/) > Wallet order
```
Comment (FrancescaNCook) [inline content/payment-methods/testing.md:548]:
```suggestion
1. [Create an order](/reference/createorder/) > Wallet order.
```

File: content/payment-methods/testing.md [hunk 6]
```diff
@@ -541,6 +541,19 @@ Any other card number receives an "Invalid card number" error.
...
+<details id="amazon-pay">
+<summary>How to test Amazon Pay</summary>
+<br>
+
+1. [Create an order](/reference/createorder/) > Wallet order  
+    Example: Amazon Pay direct/redirect
+2. On the **Test platform** page, wait for 5 seconds or click **Amazon Pay**.
+3. From the **Test scenario** list, select **Completed**.
+4. Click **Test**.  
+    The payment is processed in your MultiSafepay test account as **Successful**, with <<glossary:order status>> **Completed**, and transaction status **Initialized**.
```
Comment (FrancescaNCook) [inline content/payment-methods/testing.md:553]:
```suggestion
    The payment is processed in your MultiSafepay test account as **Successful**, with <<glossary:order status>> **Completed**, and <<glossary:transaction status>> **Initialized**.
```

File: static/diagrams/src/amazon-pay-payment-flow.mmd [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+    participant Me as Merchant
+
+    C->>Mu: Selects Amazon Pay at checkout
+    alt Redirect flow
+    Mu->>C: Redirects to payment page
+    C->>Mu: Confirms order
+    else Direct flow
+    Mu->>A: Sends payment request
+    end
+    Mu->>C: Redirects to Amazon Pay for login and payment, <br> then redirects to your success page
```
Comment (FrancescaNCook) [inline static/diagrams/src/amazon-pay-payment-flow.mmd:15]:
```suggestion
    Mu->>C: Redirects to Amazon Pay
    C->>A: Signs in and completes payment
```

File: content/payment-methods/amazon-pay.md [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+
+# Payment flow
+
+This diagram shows the flow for a successful transaction. Click to magnify.
+
+<img src="https://raw.githubusercontent.com/MultiSafepay/docs/master/static/diagrams/svg/amazon-pay-payment-flow.svg" alt="Amazon Pay payment flow" style="display: block;
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 750px;width: 100%;">
+
```
Comment (FrancescaNCook) [inline content/payment-methods/amazon-pay.md:34]:
```suggestion

> ℹ **Note** 
> MultiSafepay does **not** collect funds for Amazon Pay transactions.
```

File: content/payment-methods/chargebacks.md (hunk 3 above)
Comment (Sarah-MultiSafepay) [reply content/payment-methods/chargebacks.md:38]: Amazon specifically says in their Docs "Amazon Payments merchant account" so I'll apply the suggestion bar that.

File: content/payment-methods/google-pay-integration.md (hunk 4 above)
Comment (Sarah-MultiSafepay) [reply content/payment-methods/google-pay-integration.md:48]: No, it's all lower case.

File: content/payment-methods/amazon-pay-integration.md [hunk 9]
```diff
@@ -0,0 +1,157 @@
...
+```html
+<div id="AmazonPayButton"></div>
+```
+
+**Note:** This element is populated in a later step. For more information, see [Display the Amazon Pay button](#3-display-the-amazon-pay-button) below.
+
+
+# 2. Create an order
+
+From your server, [create an order](/reference/createorder/) > Wallet order. In the Request pane, see **Examples** > **Amazon Pay direct**s.
```
Comment (FrancescaNCook) [inline content/payment-methods/amazon-pay-integration.md:52]:
```suggestion
From your server, [create an order](/reference/createorder/) > Wallet order. In the Request pane, see **Examples** > **Amazon Pay direct**.
```

File: content/payment-methods/amazon-pay.md [hunk 10]
```diff
@@ -0,0 +1,121 @@
...
+<img src="https://raw.githubusercontent.com/MultiSafepay/MultiSafepay-icons/master/methods/amazonpay.svg" width="150" align="right" style="margin: 20px; max-height: 100px"/>
+
+<a href="https://pay.amazon.eu/" target="_blank">Amazon Pay</a> <i class="fa fa-external-link" style="font-size:12px;color:#8b929e"></i> is a payment method that Amazon customers can use to purchase goods on other sites using their Amazon account. Amazon customers can use any of their stored addresses or payment methods, such as credit or debit cards.
+
+<!-- Read how Amazon Pay can benefit your business on <a href="https://www.multisafepay.com/solutions/payment-methods/amazon-pay" target="_blank">multisafepay.com</a> <i class="fa fa-external-link" style="font-size:12px;color:#8b929e"></i> -->
+
+| Supports | Details |
+|---|---|
+| [Countries](/docs/payment-methods#payment-methods-by-country) | Austria, Belgium, Cyprus, Denmark, France, Germany, Hungary, Ireland, Italy, Luxembourg, Netherlands, Portugal, Spain, Sweden, and Switzerland. |
```
Comment (FrancescaNCook) [inline content/payment-methods/amazon-pay.md:18]:
```suggestion
| [Countries](/docs/payment-methods#payment-methods-by-country) | Austria, Belgium, Cyprus, Denmark, France, Germany, Hungary, Ireland, Italy, Luxembourg, Netherlands, Portugal, Spain, Sweden, and Switzerland | 
```

**Review by FrancescaNCook — APPROVED** [review]

File: content/payment-methods/amazon-pay.md [hunk 11]
```diff
@@ -0,0 +1,121 @@
...
+- Prestashop 1.7
+- Prestashop 1.6
+- Shopware 5
+- Shopware 6
+- WooCommerce
+
+-->
+
+### Testing
+To test Amazon Pay payments, see [Testing](/docs/testing#wallets).
```
Comment (FrancescaNCook) [inline content/payment-methods/amazon-pay.md:110]:
```suggestion
To test Amazon Pay payments, see Testing – [Wallets](/docs/testing#wallets).
```

**Pull request merged**